bigtable/examples/run_examples_utils.sh

```diff
@@ -87,51 +88,59 @@ function run_all_table_admin_examples() {
+  local -r TABLE="sample-table"
+
   # Create an instance to run these examples.
   ../examples/bigtable_samples_instance_admin create-instance "${project_id}" "${INSTANCE}" "${zone_id}" || ${ignore_unimplemented}
+  trap '../examples/bigtable_samples_instance_admin delete-instance "${project_id}" "${INSTANCE}"' EXIT
 
   echo
   echo "Run create-table example."
-  ../examples/bigtable_samples create-table "${project_id}" "${INSTANCE}" sample-table
+  ../examples/bigtable_samples create-table "${project_id}" "${INSTANCE}" "${TABLE}"
 
   echo
   echo "Run list-tables example."
-  ../examples/bigtable_samples list-tables "${project_id}" "${INSTANCE}" sample-table
+  ../examples/bigtable_samples list-tables "${project_id}" "${INSTANCE}" "${TABLE}"
 
   echo
   echo "Run get-table example."
-  ../examples/bigtable_samples get-table "${project_id}" "${INSTANCE}" sample-table
+  ../examples/bigtable_samples get-table "${project_id}" "${INSTANCE}" "${TABLE}"
 
   # Populate some data on the table, so the next examples are meaningful.
-  ../examples/bigtable_samples bulk-apply "${project_id}" "${INSTANCE}" sample-table
+  ../examples/bigtable_samples bulk-apply "${project_id}" "${INSTANCE}" "${TABLE}"
 
   echo
   echo "Run modify-table example."
-  ../examples/bigtable_samples modify-table "${project_id}" "${INSTANCE}" sample-table
+  ../examples/bigtable_samples modify-table "${project_id}" "${INSTANCE}" "${TABLE}"
 
   echo
   echo "Run drop-rows-by-prefix example."
-  ../examples/bigtable_samples drop-rows-by-prefix "${project_id}" "${INSTANCE}" sample-table
-  ../examples/bigtable_samples scan "${project_id}" "${INSTANCE}" sample-table
+  ../examples/bigtable_samples drop-rows-by-prefix "${project_id}" "${INSTANCE}" "${TABLE}"
+  ../examples/bigtable_samples scan "${project_id}" "${INSTANCE}" "${TABLE}"
 
   echo
   echo "Run drop-all-rows example."
-  ../examples/bigtable_samples drop-all-rows "${project_id}" "${INSTANCE}" sample-table
-  ../examples/bigtable_samples scan "${project_id}" "${INSTANCE}" sample-table
+  ../examples/bigtable_samples drop-all-rows "${project_id}" "${INSTANCE}" "${TABLE}"
+  ../examples/bigtable_samples scan "${project_id}" "${INSTANCE}" "${TABLE}"
 
   echo
   echo "Run delete-table example."
-  ../examples/bigtable_samples delete-table "${project_id}" "${INSTANCE}" sample-table
+  ../examples/bigtable_samples delete-table "${project_id}" "${INSTANCE}" "${TABLE}"
 
-  # Delete the instance at the end examples.
+  trap - EXIT
   ../examples/bigtable_samples_instance_admin delete-instance "${project_id}" "${INSTANCE}" || ${ignore_unimplemented}
 }
 
 function run_all_data_examples {
+  if [ ! -x ../examples/bigtable_samples ]; then
+    echo "Will not run the examples as the examples were not built"
+    return
+  fi
+
   local project_id=$1
-  shift
-  local zone_id=$1
-  shift
+  local zone_id=$2
+  shift 2
 
   local ignore_unimplemented="/bin/false"
   if [ ! -z "${BIGTABLE_EMULATOR_HOST:-}" ]; then
```

Comment:
FWIW, the shorter equivalent of `if [ ! -z "..." ]` is `if [ -n "..." ]` .

Reply:
Thanks! Fixed.